meshroom/nodes/aliceVision/FeatureExtraction.py

```diff
@@ -6,7 +6,7 @@
 class FeatureExtraction(desc.CommandLineNode):
     commandLine = 'aliceVision_featureExtraction {allParams}'
     size = desc.DynamicNodeSize('input')
-    parallelization = desc.Parallelization(blockSize=40)
+    parallelization = desc.Parallelization(blockSize=10)
```

Comment:
Not sure if it's a good idea to reduce that

Reply:
This dramatically reduced computation time on the whole panoramas dataset. I think this is fitted to the large input use case ?